resources/i18n.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, lang: str, validate: bool = False):
             if validate:
                 raise ValueError('Cannot validate book for lang %s, as resources/lang/%s.json does not exist' % (lang, lang))
             print('Writing default translation for language %s to %s' % (self.lang, self.lang_path))
-            with open(self.lang_path, 'w', encoding='utf-8') as f:
+            with open(self.lang_path, 'w+', encoding='utf-8') as f:
```

Comment:
No? We aren't reading from it here.

Reply:
Oh whoops.

TIL that's what w+ means. I thought w meant write to an existing file, and w+ meant write to an existing file, or create a new file if it doesn't exist.

My bad—nevermind